docs/docs/uninstall.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Manual Uninstall
+---
+
+------------
+
+**DISCLAIMER**
+_This is a very early version of Cassandra-on-Mesos framework. This document, code behavior, and anything else may change without notice and/or break older installations._
+
+------------
+
+
+# Manual Uninstall on DCOS
+
+To Uninstall the Cassandra Mesos Framework cleanup the following items:
+
```

Comment:
Worth pointing out that this will _completely_ uninstall the Cassandra-on-Mesos framework - data and all.